### PR TITLE
526 bdd messaging

### PR DESCRIPTION
**Pull request by cvalarida**
## Description
Lotsa messaging to dissuade veterans to fill out the form if they're currently in the service. Instead, we redirect them to eBenefits, where they may be able to.

## Testing done
Wrote unit tests and made sure it worked locally like I expected it to.

### Are you sure this is right?
image

### Turn back, Sarah! Before it's too late!
image

### Separation date < 90 days away
image

### Separation date between 90 & 180 days away
image

### Separation date > 180 days away
image

## Acceptance criteria
- [x] The BDD verification page is displayed if there are active service periods
- [x] The BDD verification page displays all active service periods ("all" in the weird event that there's more than one)
- [x] When the user selects "no, that's not right, " they're shown the "go back" page
- [x] The "go back" page links back to the military history page
- [x] When the user selects "yes, that's right," they're shown some version of the "sorry, we can't do that here" page
- [x] The "sorry, we can't do that here" page informs the user of the next steps depending on their separation date
    - [x] < 90 days; redirect to eBenefits with a message stating that BDD isn't possible, but they can get started
    - [x] Between 90 & 180 days; redirect to eBenefits with a message about BDD
    - [x] > 180 days; say sorry

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/disability-benefits/all-claims/config/form.js [hunk 1]
```diff
@@ -83,10 +85,11 @@ import {
   ptsdWalkthroughChoice781a,
   ratedDisabilities,
   reservesNationalGuardService,
+  retirementPay,
```
Comment (cvalarida) [inline src/applications/disability-benefits/all-claims/config/form.js]: Decided to run `sort` on these imports. :man_shrugging:

File: src/applications/disability-benefits/all-claims/config/form.js [hunk 2]
```diff
@@ -186,6 +189,27 @@ const formConfig = {
           onContinue: captureEvents.militaryHistory,
           appStateSelector: state => ({ dob: state.user.profile.dob }),
         },
+        verifyBdd: {
+          title: 'Verify active duty status',
+          path: 'review-veteran-details/verify-bdd',
```
Comment (cvalarida) [inline src/applications/disability-benefits/all-claims/config/form.js]: @katelyncai @goldenmeanie Do you have opinions about the URLs to these three new pages?

File: src/applications/disability-benefits/all-claims/pages/index.js [hunk 3]
```diff
@@ -23,8 +25,8 @@ import * as incidentLocation from './incidentLocation';
 import * as incidentSupport from './incidentSupport';
 import * as incidentUnitAssignment from './incidentUnitAssignment';
 import * as incomeDetails from './incomeDetails';
-import * as individualsInvolved from './individualsInvolved';
 import * as individualsInvolvedFollowUp from './individualsInvolvedFollowUp';
+import * as individualsInvolved from './individualsInvolved';
```
Comment (cvalarida) [inline src/applications/disability-benefits/all-claims/pages/index.js]: Ran `sort` on these too... :sweat_smile:

File: src/applications/disability-benefits/wizard/pages/bdd.jsx [hunk 4]
```diff
@@ -27,16 +28,11 @@ const alertContent = (
       file a BDD claim, but you can still begin the process of filing your claim
       on eBenefits.
     </p>
-    <a
-      href="https://www.ebenefits.va.gov/ebenefits/about/feature?feature=disability-compensation"
```
Comment (cvalarida) [inline src/applications/disability-benefits/wizard/pages/bdd.jsx]: I figured if we're going to link to this in multiple places, we may as well make it a constant.

**Review by peggygannon — CHANGES_REQUESTED** [review]
@cvalarida 
Here are some tweaks for that last message: 
- [ ] Please rewrite to: 
**Your separation date is more than 180 days away**, so you can't file a disability claim. Please check back when your separation date is less than 180 days away. 

- [ ] please remove the link to [Learn more about the BDD program] in this alert box.

**Comment by cvalarida** [comment]
@peggygannon Sounds good. What should the header content for the alert box say? @markgreenburg pointed out that it probably shouldn't say "file on eBenefits" if they can't actually do that.

**Comment by peggygannon** [comment]
@cvalarida Yes, thank you! I've updated the header and tweaked the message some more. We also should remove the intro line. 

here are my proposed changes: 
- [ ] Remove the sentence: You're still in the service, so you won't be able to submit this application here on VA.gov. 
- [ ] Update alert header to: 
**You can't file a disability claim right now** 
You can't file a disability claim until 180 days before you leave the service. Please check back when your separation date is less than 180 days away. In the meantime, you can begin to gather any supporting documents or evidence you'll submit for your claim.

**Review by bshyong — APPROVED** [review]
LGTM

File: src/applications/disability-benefits/all-claims/content/bddRedirect.jsx [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+  const endDates = activeServicePeriods(formData).map(
+    sp =>
+      sp.dateRange.to ? moment(sp.dateRange.to) : moment().add(200, 'days'),
+  );
+  const oneEighty = moment().add(180, 'days');
+  const ninety = moment().add(90, 'days');
+
+  let headline = 'You’ll need to file your claim on eBenefits';
+  let content;
+  const eBenLink = (
```
Comment (hinzed1127) [inline src/applications/disability-benefits/all-claims/content/bddRedirect.jsx]: probably overkill to create utility components for this and the learn more link unless we think we'll use them in letters more than the 2 instances in this PR

**Review by hinzed1127 — APPROVED** [review]

File: src/applications/disability-benefits/all-claims/content/bddRedirect.jsx (hunk 5 above)
Comment (cvalarida) [reply src/applications/disability-benefits/all-claims/content/bddRedirect.jsx]: :thinking: I'm not sure. I _think_ the most important part is making the URL itself shared. Once we have that, we can display it however we want—either as a normal link or a "button" link without having to worry about changing it anywhere else it's used if we update the style later.

:man_shrugging:

**Review by peggygannon — APPROVED** [review]

**Pull request merged**